core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments/activity_items_controller.rb

```diff
@@ -4,6 +4,8 @@ class UffizziCore::Api::Cli::V1::Projects::Deployments::ActivityItemsController
   UffizziCore::Api::Cli::V1::Projects::Deployments::ApplicationController
   before_action :authorize_uffizzi_core_api_cli_v1_projects_deployments_activity_items
 
+  rescue_from UffizziCore::DeploymentStateError, with: :render_deployment_state_exception
```

Comment:
I think it's over complicated. Let's do simple as possible.
In that case we can just write something like this:
```
  deployment = resource_deployment.existed
  if deployment.failed?
    return render json: { errors: { title: [I18n.t '...'] } }, status: :unprocessable_entity
  end

    activity_items = deployment
      .activity_items
      .page(page)
      .per(per_page)
      .order(updated_at: :desc)
      .ransack(q_param)
      .result
```

And let's use I18 everywhere possible